uxarray/grid/grid.py

```diff
@@ -1749,6 +1751,15 @@ def get_kd_tree(
 
         return self._kd_tree
 
+    def get_spatialhash(
+        self,
+        reconstruct: bool = False,
+    ):
```

Comment:
Please include a docstring, similar to how `get_kd_tree()` and `get_ball_three()` are formatted.

Reply:
Looks great!